omegaconf/dictconfig.py

```diff
@@ -117,6 +117,9 @@ def __copy__(self) -> "DictConfig":
         for k, v in self.__dict__.items():
             res.__dict__[k] = copy.copy(v)
         res._re_parent()
+        for k, v in self.__dict__["_content"].items():
+            if isinstance(v, ValueNode):
+                res.__dict__["_content"][k] = copy.copy(v)
```

Comment:
content can be None or MISSING, I think it will blow up here.

Reply:
In this case copying Foo doesn't raise and error neither with None. Since we only copy.copy ValueNode.
```python
@dataclass
class Foo:
    a: Dict[str, int] = MISSING
    b: Any = 2
```

Reply:
I think you misunderstood me.
`__dict__["_content"]` itself can be None or '???'. In that case it will not have an items() method.

Reply:
But, `self.__dict__["_content"]` can only be none or missing if it's specified in a structured config, isn't it?

Reply:
Yes. but the fact that you were not able to reproduce the problem does not mean it's not there.

if _content is None,  the line `for k, v in self.__dict__["_content"].items():` will fail, right?

Reply:
Were you able to reproduce it?
It would be good to add a test that tests serializing such DictConfigs.

Reply:
Yes, done in 8fee3099f5c958004d106ddd3d5d8cd14f8d2248

Reply:
Do I add test for both `OmegaConf.save()` and pickle?

Reply:
save is converting to yaml, right?
does it even hit that code path?

Reply:
I think pickle doesn't hit that part too.

Reply:
It doesn't copy, got it. Done in 4881890